Configuration/StandardSequences/python/Validation_cff.py

```diff
@@ -103,3 +103,23 @@
 from Configuration.Eras.Modifier_fastSim_cff import fastSim
 fastSim.toReplaceWith(validation_prod,_validation_prod_fastsim)
 
+from Validation.EcalClusters.ecalClustersValidationSequence_cff import ecalClustersValidationSequence
```

Comment:
@tanmaymudholkar @jfernan2 sorry, but I do not think this is the correct place where to put such a definition, why not in the specialized ECAL validation configuration https://github.com/cms-sw/cmssw/blob/master/Validation/Configuration/python/ecalSimValid_cff.py ?

Reply:
@fabiocos I do agree... and the same applies to the just merged master version of this #27549

Reply:
@jfernan2 sorry, but there is nothing merged yet, I have put that on hold pending a clarification and code rearrangment

Reply:
@jfernan2 @fabiocos : Indeed the ECAL-specific configuration fragment seems like a better place for this definition. I've put it there and tested the workflows in the latest commit.